book/src/library/dart_only.md

```diff
@@ -9,6 +9,11 @@ The following script assumes a bash shell, which you should make sure to use to
 Also, the script generates some ffi convenience files in your Dart `src/` which you should
 check out after the script completes.
 
+After the script runs, change the `flutter_rust_bridge` dependency in `/packages/library_name/pubspec.yaml` to the following:
+```yaml
+  flutter_rust_bridge: "1.62.1"
```

Comment:
uber-nit: what about telling the reader that "please replace the version to whatever the latest version is"

Reply:
I've had quite a few breaking changes in the last 10 or so FRB versions in my own package, so I am a bit hesitant to recommend users to use the latest version of FRB in the docs directly (this wouldn't be a problem with proper CI testing, but as I mentioned in #941, that might want to wait). I know that 1.62.1 works properly, which is why I put that explicitly in the docs.

I think if users want to update the FRB version after they finish the docs, that would be OK, but it could break their project. I just didn't want to put it in that part of the docs since users might wonder why the package the docs helped them create doesn't work.

Reply:
Looks reasonable! So maybe add a line saying "1.62.1 is tested to work; you may use newer versions but this part does not have proper testing yet so may breaking change"

Reply:
Should be fixed in [406ac38](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1050/commits/406ac38541f7a41a50e0365ac7f5c692779980aa)